src/views/create/createWallet.jsx

```diff
@@ -36,7 +37,7 @@ function Create({
 
         if (!isPassphraseValid(value)) {
           setPassPhraseError(
-            "Passphrase must be minimum 15 characters, 1 min uppercase letter and 1 special character."
+            "Passphrase must be minimum 8 characters, 1 min uppercase letter and 1 special character."
```

Comment:
Q: Where are the changes related to this change in length for passphrases? Did we do them in an earlier commit?

Reply:
No, it has always worked that way.